<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>年龄</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>爱好</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_junshi">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>军事</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_tiyu">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>体育</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_yule">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>310</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>娱乐</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_caijing">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>380</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>财经</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_dongman">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>380</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>动漫</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>确认注册</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_man">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_woman">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>237</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>女</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_tooyoung">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>113</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>16岁以下</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_young">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>176</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>16岁——30岁</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_old">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>359</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>30岁以上</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
